app/services/handover_date_service.rb

```diff
@@ -39,6 +39,8 @@ def self.handover(raw_offender)
       responsibility = responsibility_override(offender)
       if responsibility.nil?
         handover_date, reason = nps_handover_date(offender)
+        start_date = nps_start_date(offender)
+        handover_date = start_date if start_date > handover_date
         HandoverData.new nps_responsibility(offender, handover_date), nps_start_date(offender), handover_date, reason
```

Comment:
Optimisation: you should be able to avoid the second call to `nps_start_date(offender)` in the final line (`HandoverData.new...`) because you already hold the calculated start date in the variable `start_date`